docker/conf/nginx.conf

```diff
@@ -60,9 +63,11 @@ http {
     server_name         localhost;
     access_log          off;
 
+		set $kibana_proxy_pass_url http://127.0.0.1:5601;
```

Comment:
Same here, I think tabs vs spaces issue perhaps?

Reply:
Maybe, I'll see if I can figure out what's going on between my ide and me, will correct shortly.

Reply:
I know what is causing this. They are set to spaces in the IDE, and are indented correctly there, but because I submitted the commit from my windows machine git is trying to correct for crlf I think that may be throwing things off.

Reply:
I just pulled it down on the mac, and indentation is correct there too for these files.
Whatever github is showing is incorrect.